Libs/MRML/Core/vtkMRMLVolumeArchetypeStorageNode.cxx

```diff
@@ -281,6 +282,39 @@ void ApplyImageSeriesReaderWorkaround(vtkMRMLVolumeArchetypeStorageNode * storag
     }
   }
 }
+
+//----------------------------------------------------------------------------
+bool IsVolumeCoordinateSystemLeftHanded(const vtkSmartPointer<vtkMatrix4x4>& ijkToRASMatrix)
```

Comment:
To clarify which coordinate system, use IJK:
```suggestion
bool IsIJKCoordinateSystemLeftHanded(const vtkSmartPointer<vtkMatrix4x4>& ijkToRASMatrix)
```

Reply:
Thanks, I renamed the method as per your suggestions and renamed the input to rasToIjkMatrix for consistency with the call.